alacritty_terminal/src/ansi.rs

```diff
@@ -316,6 +316,12 @@ pub trait Handler {
 
     /// Pop the last title from the stack.
     fn pop_title(&mut self) {}
+
+    /// Report window size in pixels.
```

Comment:
I used “window size” because it is the terminology used in other terminal implementations.

It is changed to “text area size” in c76d20529d00ee4b31bb8a752d313822c16a31d8.